packages/browserify/test/globalRef.js

```diff
@@ -114,9 +23,84 @@ test('globalRef - has only the expected global circular refs', async (t) => {
       'self',
       'global',
       'globalThis'
-    ],
+    ].sort(),
+    testType: 'truthy'
+  })
+
+  scenario.checkResult = async (t, result, scenario) => {
+    result.sort()
+    if (scenario.testType === 'truthy') {
+      t.assert(Array.isArray(result), `${scenario.name} - scenario gives expected truthy result`)
+    } else if (scenario.testType === 'falsy') {
+      t.falsy(result, `${scenario.name} - scenario gives expected falsy result`)
+    } else {
+      t.deepEqual(result, scenario.expectedResult, `${scenario.name} - scenario gives expected result`)
+    }
+  }
+  await runAndTestScenario(t, scenario, runScenario)
+  scenario.testType = 'truthy'
+  await runAndTestScenario(t, scenario, runScenario)
```

Comment:
- runAndTestScenario only once
- instead of overridding checkResult just sort the result inside the `defineOne`
- test type should not be truthy, should be testing for equality

Reply:
It's a bit tricky to read, but the first `runAndTestScenario` tests for equality, the second one tests to make the sure the result is an array. That's why I modified checkResult. Let me know if it's too convoluted

Reply:
just test normally once to see if the test result matches expected (the sorted arrays)